### ISSUE TITLE
importing NFE

### ISSUE THREAD

**Issue by Chechgm** [freelunchtheorem/Conditional_Density_Estimation#8]
Hey,

This is a bit of a silly question. 
**A bit of background**: I am trying to implement a normalizing flows cde on pytorch but unfortunately my implementation is not working. However, I am not 100% sure it is because of the data I am simulating. Particularly, I am simulating rotations of sklearn's half moon data set with a small variance change. This is, my "x" are 2d one first dimension is a normalized angle and the second dimension is a normalized variance. My "y" on, the other hand are just the coordinates of the half moon generated by sklearn. This means there is a very small variance in my x (there are only 10 pairs, each one corresponding to 300 coordinates of the moons data set).

 **Now the actual problem**: My implementation does not seem to work and I wanted to check with your package whether is a problem on the data or of my implementation but I cant seem to import the normalizing flows estimator. If you could show me how to do it or point me to the section in the docs where this is specified (I didnt find it myself) I'd be super grateful.

Best,
Sergio

PS: for two angles and two variances my data set looks like this: 
![halfmoons](https://user-images.githubusercontent.com/7306824/62423933-e5b37000-b6c7-11e9-9e75-6b737acacd8e.png)
and the resulting density estimation for a specific angle and a specific value of the variance is :
![results](https://user-images.githubusercontent.com/7306824/62423936-f6fc7c80-b6c7-11e9-9459-e7bbad685787.png)
Which is obviously wrong. I am using what I believe is enough power on the NN, one or two hidden layers with 32 hidden units and 10-15 flows.

**Comment by siboehm** [comment]
We haven't yet updated the PyPi package to include the Normalizing Flow Estimator, to be able to import the NFE you'll have to build the package from source. `pip install .` after cloning.

I can't really speak about whether or not your data generation or NF implementation is correct. 10 flows should be easily able to fit two-moons. Your last plot is difficult to read, but if the density estimate is the grey background I'd find it suspicious that is seems to be so consistent over y.

**Comment by Chechgm** [comment]
Yes, the grey "background" is the density estimation. Thank you for your response, Whenever I have time I will build it from source.
